admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -75,42 +101,54 @@ export const ApplicationPage: React.FunctionComponent = () => {
                         <Toolbar>
                             <ToolbarContent>
                                 <ToolbarItem>
-                                    <Button variant='primary' type='button' onClick={ toggle }> Create Event Type </Button>
+                                    <Button variant='primary' type='button'
+                                        onClick={ createEventType }> Create Event Type </Button>
                                     <Modal
                                         variant={ ModalVariant.medium }
                                         title={ `Create Event Type for ${ (applicationTypesQuery.loading ||
```

Comment:
The `isEdit` (or `visable`) could help here to find out if we are Creating an event type or Updating an event type.

Reply:
I think this is the only remaining ask I have here.
Could you conditionally change the title?
something like: `${isEdit ? 'Update' : 'Create'}` instead of the `Create`

Reply:
of course!